common-compose.yml

```diff
@@ -1,35 +1,39 @@
 # MySQL designed for container-native deployment on Joyent's Triton platform.
 
-mysql:
-    image: autopilotpattern/mysql:latest
-    mem_limit: 4g
-    restart: always
-    env_file: _env
-    environment:
-      - CONTAINERBUDDY=file:///etc/containerbuddy.json
-      - MANTA_PRIVATE_KEY
+version: '2'
 
-    # expose for linking, but each container gets a private IP for
-    # internal use as well
-    expose:
-      - 3306
+services:
 
-# ---------------------------------------------------
-# Consul as a service discovery tier
+    mysql:
+        image: autopilotpattern/mysql:latest
+        mem_limit: 4g
+        restart: always
+        env_file: _env
+        environment:
+            - CONTAINERBUDDY=file:///etc/containerbuddy.json
```

Comment:
Any particular reason we don't just embed `CONTAINERBUDDY=file:///etc/containerbuddy.json` directly in the image itself?